README.md

```diff
@@ -31,6 +31,9 @@ yarn storybook
 yarn build-storybook
 ```
 
+The `main` branch automatically gets deployed to https://satellytes-website-storybook.netlify.app. Pull Requests on 
```

Comment:
Maybe we can make it more clear that this is about storybook and not the gatbsy application itself? I do realize that this is below the headline "Component development with Storybook" and that the URL contains `storybook` - but I still think we should mention it in the plain text.
Something like: "The storybook for the `main` branch gets deployed ..."?